playlistmanager.lua

```diff
@@ -209,6 +194,44 @@ local requested_urls = {}
 --state for if we sort on playlist size change
 local sort_watching = false
 
+local filetype_lookup = {}
+
+function update_opts(changelog)
+  msg.verbose('updating options')
+
+  --parse filename json
+  if changelog.filename_replace then
+    if(settings.filename_replace~="") then
+      settings.filename_replace = utils.parse_json(settings.filename_replace)
+    else
+      settings.filename_replace = false
+    end
+  end
+
+  --parse loadfiles json
+  if changelog.loadfiles_filetypes then
+    settings.loadfiles_filetypes = utils.parse_json(settings.loadfiles_filetypes)
```

Comment:
We should reset the `filetype_lookup` variable here before the for loop, otherwise the changes are not overwriting the previous setting.

Reply:
You're the expert. I can do this when I get time tomorrow. Or you could edit it directly, you should have access.

Reply:
Ok, just figuring out how to do that, since I haven't worked a lot with github pull requests.